packages/react-date-time/src/components/DatePicker/examples/DatePicker.md

```diff
@@ -0,0 +1,36 @@
+---
+id: Date picker
+section: components
```

Comment:
Shouldn't this be listed under extensions? If we're fully committing to it as a component, we should add a message like we do with Table:
```md
Note: Table lives in its own package at [@patternfly/react-table](https://www.npmjs.com/package/@patternfly/react-table)!
```